demo/src/sampleTrace.js

```diff
@@ -70,7 +70,11 @@ export function sampleTrace() {
             serviceName: 'Methods',
             color: "#ff0000",
             tags: {
-                error: true
+                error: true,
+                'method.startLine': 158,
```

Comment:
What are these, can we add comments explaining the use cases? What is the magical number 158?